src/components/Code/Code.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import { isFunction } from 'lodash';
+
+const CopyNotice = styled.div`
+  position: absolute;
+  top: 5px;
+  right: 5px;
+  transition: opacity 300ms ease;
+  opacity: 0;
+  font-size: ${props => props.theme.fontSizes.tiny};
+  color: ${props => props.theme.colors.athensGray};
+`;
```

Comment:
I suggest a couple of styling changes here that IMO make it look better:

* Set both `top` and `right` to `0`
* Add padding `10px 15px` instead, like in the wrapper element
* Add the same `borderRadius` and `backgroundColor` like the wrapper element as well
* Set the full opacity to `0.9` instead of `1` below, so that the text overlay over the code looks a bit better
* Set font size to `small` instead of `tiny` after all

Basically, I think the text shouldn't be that much stuck to the corner and increasing the padding will potentially cause overlap with the text (which should be fine), therefore the rest of the changes.

cc @bia